barbican/cmd/barbican_manage.py

```diff
@@ -24,6 +24,9 @@
 
 from oslo_config import cfg
 from oslo_log import log as logging
+from oslo_config import cfg as oslo_cfg
+
+from sqlalchemy import create_engine, text
```

Comment:
For Nova, upstream only likes one import per line. Did you run the pep8 check of barbican on this file?